src/validations/rules/sar.sch

```diff
@@ -178,47 +242,91 @@
+                value="
+                    for $i in $combined
+                    return
+                        $resolved-profile-doc//oscal:control[@id = $i]//oscal:part[@name = 'objective'][oscal:prop[@name = 'response-point']]/@id" />
+            <!-- targetid values of the findings of the most recent result -->
+            <sch:let
+                name="first-result-finding-target-IDs"
+                value="//oscal:result[@uuid = $first-result]//oscal:finding//oscal:target[@type = 'objective-id']/@target-id" />
+            <!-- matching profile control ids with first result finding target ids -->
+            <sch:let
+                name="target-IDs-combined"
+                value="$matching-control-ids[. = $first-result-finding-target-IDs]" />
+            <!-- profile control ids that are not in the first result finding target ids -->
+            <sch:let
+                name="target-IDs-combined-missing"
+                value="$matching-control-ids[not(. = $first-result-finding-target-IDs)]" />
+
+            <sch:assert
+                diagnostics="not-in-resolved-baseline-controls-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAR) §4.2"
+                id="not-in-resolved-baseline-controls"
+                role="error"
+                test="
+                    if (@uuid = $first-result)
+                    then
+                        count($not-in-resolved-baseline-controls) = 0
+                    else
+                        true()"
+                unit:override-xspec="both">All the in scope controls must exist in the resolved baseline profile document.</sch:assert>
 
+            <sch:assert
+                diagnostics="not-target-ids-in-sar-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAR) §4.2"
+                id="not-target-ids-in-sar"
+                role="error"
+                test="
+                    if (@uuid = $first-result)
+                    then
+                        count($target-IDs-combined-missing) = 0
+                    else
+                        true()"
+                unit:override-xspec="both">All in scope controls in the resolved baseline profile must have objectives in the most recent result
+                assembly that match a finding with a target/@target-id.</sch:assert>
+
+                <sch:assert
+                    diagnostics="has-attestation-diagnostic"
+                    fedramp:specific="true"
+                    id="has-attestation"
+                    role="error"
+                    test="oscal:attestation[oscal:part[@name = 'authorization-statements']/oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'recommend-authorization']]">
+                    There must exist an attestation with a part containing a property with a name of 'recommend-authorization'.</sch:assert>
```

Comment:
This is missing a documentation reference - it seems to be from §4.12 of the guide.